tests/dags/test_only_dummy_tasks.py

```diff
@@ -29,14 +29,24 @@
 
 dag = DAG(dag_id="test_only_dummy_tasks", default_args=default_args, schedule_interval='@once')
 
+
```

Comment:
This DAG is used by `test_should_mark_dummy_task_as_success` test in `tests/jobs/test_scheduler_job.py`